geometry_msgs/Pose2D task_data
string task_type
int64 task_id
